### ISSUE TITLE
Can not convert struct creating by golang package

### ISSUE THREAD

**Issue by allanassis** [callmehiphop/pb-util#9]
Hi, I am trying to use the `decode` method from `pb-util` and I am having an issue.
I have a grpc server written in golang and it returns to me a message in this format.

```
{
  "values": {
    "fields": {
      "url": {
        "stringValue": "url-da-receita"
      },
      "nested": {
        "structValue": {
          "fields": {
            "foo": {
              "stringValue": "bar"
            }
          }
        }
      }
    }
  },
  "message": "Ok"
}

```

I am trying to convert this message using your package but seems not to work.
I don't know why but seems the structure is a bit different because this message generated does not have the `kind` property.

**Comment by callmehiphop** [comment]
👋 @allanassis - thanks for the bug report! I went ahead and prepared a branch (https://github.com/callmehiphop/pb-util/pull/10) with a potential fix, would you be able to give it a test run to see if resolves your issues?

**Comment by allanassis** [comment]
Thanks, @callmehiphop! It works!

Thanks for solving my problem so quickly.

**Comment by callmehiphop** [comment]
@allanassis that's actually a really good question! I think that `kind` has always been optional when creating request messages, so maybe it is just optional everywhere. 🤷 

Anyway, I'm going to do a little housekeeping, but I should have a release with this fix cut today. Thanks again for the bug report!

**Comment by callmehiphop** [comment]
Just published `v1.0.0`, if you spot any regressions,etc., please let me know! Thanks again!
